### PR TITLE
Integrate Cypress

### PR DESCRIPTION
**Pull request by rexagod**
Adds cypress support for running e2e tests in an OCP cluster.

Signed-off-by: Pranshu Srivastava <rexagod@gmail.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,7 +51,10 @@
     "react-tagsinput": "^3.19.0"
   },
   "devDependencies": {
-    "@openshift-console/dynamic-plugin-sdk": "0.0.3",
+    "@cypress/webpack-preprocessor": "^5.9.1",
+    "cypress": "^8.3.1",
+    "cypress-multi-reporters": "^1.5.0",
+    "@openshift-console/dynamic-plugin-sdk": "0.0.4",
```
Comment (rexagod) [inline package.json:57]: @SanjalKatiyar In the older PR diff, I see that the version used was 0.0.4, but now its 0.0.3: https://github.com/red-hat-storage/mcg-ms-console/pull/6/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R48. Is this intentional? If so, I'll revert this change.

**Comment by rexagod** [comment]
@naveenpaul1 PTAL https://github.com/red-hat-storage/mcg-ms-console/pull/6#discussion_r868000475
@v-harihar PTAL https://github.com/red-hat-storage/mcg-ms-console/pull/6#discussion_r868001888

File: package.json (hunk 1 above)
Comment (rexagod) [reply package.json:57]: Is the test failing because of this?

**Comment by rexagod** [comment]
Screenshot from 2022-05-17 22-03-17

File: test-prow-e2e.sh [hunk 2]
```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env bash
+
+set -eExuo pipefail
+
+NAMESPACE="redhat-data-federation"
+MCG_MS_CONSOLE_IMAGE="" # TODO
```
Comment (SanjalKatiyar) [inline test-prow-e2e.sh:6]: just want to confirm before merging... what are we planning for this ?

**Pull request merged**